### ISSUE TITLE
Review README

### ISSUE THREAD

**Issue by jlengstorf** [apache/openwhisk-website#38]
I haven't touched the README from the original OpenWhisk site. Does this need to be updated with Apache branding, new license, etc.?

**Comment by krook** [comment]
May also need CONTRIBUTING/CLA info.

**Comment by krook** [comment]
We should also add the Jekyll preview steps:


## Quickstart for Local Development

``` sh
# Clone the repository (and correct branch).
git clone https://github.com/openwhisk/openwhisk.github.io.git

# Move into the cloned repo.
cd openwhisk.github.io

# Start the development server.
bin/develop
```

**Comment by jlengstorf** [comment]
@krook Added. :)

**Comment by nauerz-ibm** [comment]
Looks good to me, except that I wonder whether we cannot simply use the same phrasing to explain what OpenWhisk is we have used before: "OpenWhisk is a serverless open-source cloud platform that allows you to execute code in response to events at any scale and without the need to manage servers. I.e. that it provides you with a serverless deployment and operations model hiding infrastructural complexity."
Just to be consistent.

Also, for pointers to other communication channels I recommend we mention all channels we have:
I would just go through the icons we added to the very top of the page:
Git, Youtube, Slideshare, Twitter, Stackoverflow, Slack, Medium, ...
